config-linux.md

```diff
@@ -622,7 +622,14 @@ The following parameters can be specified to set up seccomp:
 ## <a name="configLinuxRootfsMountPropagation" />Rootfs Mount Propagation
 
 **`rootfsPropagation`** (string, OPTIONAL) sets the rootfs's mount propagation.
-    Its value is either slave, private, shared or unbindable.
+    Its value is either `shared`, `slave`, `shared+slave`, `private` or `unbindable`. It's worth noting that a peer group is defined as a group of VFS mounts that propagate events to each other.
+
+  * `shared`: the rootfs mount belongs to a new peer group. This means that further mounts (e.g. nested containers) will also belong to that peer group and will propagate events to the rootfs. Note this does not mean that it's shared with the host.
```

Comment:
nits: these should use four-space intends (#495), and use [new lines for each sentence][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/style.md#one-sentence-per-line

Reply:
@wking Good to know. Thanks!